### ISSUE TITLE
Bug: hostnames not showing up

### ISSUE THREAD

**Issue by toorusr** [mwarning/device-observatory#13]
The hostnames of all other devices then mine are no showing up

**Comment by mwarning** [comment]
Actually, that is a privacy feature. If you access the program from localhost, you should be able to see all data. Or export all data to json (`--json-output`).
I intended to make it configurable, but didn't come around to do it.

**Comment by mwarning** [comment]
In case you already run the program on localhost, then this is indeed a bug. :-)

**Comment by mwarning** [comment]
I just tested it. `./device-observatory --dev eth0` and then I went to localhost:800, clicked on one of the detected MAC addresses of my computer and got host names displayed.

Maybe you clicked on entries of devices on the network, that do not use that interface (eth0) as gateway. In that case you get a bunch of rather plain entries.
Usually device-observatory is meant to run on the gateway (e.g. wifi router).
